mds/apis/agency_api/v0_x/compliance.py

```diff
@@ -0,0 +1,121 @@
+from rest_framework import serializers
+from rest_framework import viewsets
+
+from django.db.models import Q, Prefetch
+
+from mds import models
+
+from datetime import datetime
+import pytz
+
+
+class ComplianceSerializer(serializers.ModelSerializer):
+    rules_id = serializers.SerializerMethodField()
+
+    compliances = serializers.SerializerMethodField()
+
+    class Meta:
+        model = models.Policy
+        fields = ("id", "rules_id", "compliances")
+
+    def get_rules_id(self, policy):
+        rules = []
```

Comment:
There is a way to write the same this in one line :  
`return [rule["rule_id"]] for rule in policy.rules]`

This is what we call a one-liner

Reply:
Propre